packages/terra-avatar/src/Avatar.scss

```diff
@@ -1,3 +1,5 @@
+@import '~terra-mixins';
```

Comment:
Why did we re-add this?

Reply:
Avatar.scss uses a sass function named `inline-svg` that is provided in terra-mixins.